frontend/src/pages/tracker-report.module.scss

```diff
@@ -0,0 +1,277 @@
+@import "../styles/tokens.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+
+.wrapper {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+@keyframes fadein {
+  from {
+    opacity: 0;
+  }
+  50% {
+    // Let the loading screen stay hidden for 200ms;
+    // if it gets removed in that time (because the report has initialised),
+    // showing a "loading" message is just distracting:
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+.loading {
+  padding: $spacing-xl;
+  height: 100%;
+  background-color: $color-grey-05;
+  animation-name: fadein;
+  animation-duration: 400ms;
+}
+
+.load-error {
+  padding: $spacing-xl;
+  height: 100%;
+  background-color: $color-grey-05;
+}
+
+.report-wrapper {
+  background-color: $color-grey-05;
+  flex: 1 0 auto;
+  display: flex;
+  justify-content: center;
+
+  .report {
+    width: 100%;
+    max-width: $content-xl;
+    padding: $spacing-xl;
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-md;
+  }
+
+  .logo {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: $spacing-xs;
+
+    .logomark {
+      max-height: 35px;
+    }
+    .logotype {
+      max-height: 20px;
+    }
+  }
+
+  h1 {
+    @include text-title-2xs;
+    font-family: $font-stack-firefox;
+    padding: $spacing-md 0;
+    border-bottom: 1px solid $color-grey-20;
+  }
+
+  .meta {
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-md;
+    padding: $spacing-lg;
+
+    @media screen and #{$mq-md} {
+      flex-direction: row;
+      gap: $spacing-2xl;
+      padding: $spacing-lg $spacing-2xl;
+    }
+
+    dt {
+      color: $color-grey-40;
+    }
+
+    dd {
+      font-weight: 600;
+      color: $color-grey-50;
+    }
+
+    .from {
+      flex: 1 0 auto;
+    }
+  }
+
+  .trackers {
+    background-color: $color-white;
+    border-radius: $border-radius-md;
+    display: flex;
+    flex-direction: column;
+    box-shadow: $box-shadow-sm;
+    // We typically avoid margins because they affect the positioning of other elements,
+    // but since box-shadow is visible below the element, the bottom margin lets
+    // us pretend like it's part of the element:
+    margin-bottom: $spacing-md;
+    padding: $spacing-lg;
+    gap: $spacing-md;
+
+    @media screen and #{$mq-md} {
+      padding: $spacing-lg $spacing-2xl;
+      gap: $spacing-xl;
+    }
+
+    h2 {
+      @include text-title-xs;
+      font-family: $font-stack-firefox;
+      font-weight: 700;
```

Comment:
Correct me if I'm wrong, but I think the standard we stuck to was setting `font-weight` to `500` for bolded text?

Reply:
Are there instances of when we should be using `700` or `500`?

Reply:
Hmm, not sure, I took that from the Figma. I'll ask Eduardo about both that and the icon size.

Reply:
Actually, the icon size was my fault, fixed in 3f6a7857.

I also added the truncation for long email addresses, thanks for the suggestion: 5081a2c5

Reply:
Confirmed about the bolded text, and fixed in 400833ff.

(I see there's a lot of other instances of 700 though; I might submit a follow-up PR to fix all those once my queue of outstanding PRs is smaller :slightly_smiling_face: )